src/components/Task/List/List.tsx

```diff
@@ -434,7 +434,11 @@ const TaskList = ({ initialFilter }: Props): ReactElement => {
       setCurrentPage(newPage);
     },
     onRowClick: (_rowData, rowMeta) => {
-      openTaskDrawer({ taskId: data.tasks.nodes[rowMeta.dataIndex].id });
+      openTaskDrawer({
+        taskId: data.tasks.nodes[rowMeta.dataIndex].id,
```

Comment:
This doesn't work? (I didn't look at any docs...)
```suggestion
        taskId: rowData.id,
```

Reply:
It looks like ```rowData``` is just an array of strings

Reply:
Huh, weird 🤷‍♂️